src/gemm_v2.cpp

```diff
@@ -40,7 +40,7 @@
 
 #if MIOPEN_USE_ROCBLAS
 #include <half.hpp>
-#include <rocblas.h>
+#include <rocblas/rocblas.h>
```

Comment:
You will need to check if rocblas/rocblas.h header exists before including it. Otherwise this breaks miopen builds with older rocm versions which includes our CI.